Mvc.sln

```diff
@@ -178,6 +178,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Mvc.Ap
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RazorRendering", "benchmarkapps\RazorRendering\RazorRendering.csproj", "{D7C6A696-F232-4288-BCCD-367407E4A934}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebSiteCommon", "test\WebSites\WebSiteCommon\WebSiteCommon.csproj", "{290A289C-AE5A-4857-A62A-14B8E8C776A6}"
```

Comment:
Would it be possible to make these shared files? Is it important to the test that they be in a class library? We really don't want more projects here unless it's significant.

Reply:
In SignalR project we had a `Common` directory containing files that are included as links. Want something like that?

Reply:
Yes please